doc/_ext/techniques.json

```diff
@@ -214,6 +219,11 @@
     "Alternative names": "X-ray Natural Circular Dichroism",
     "Description": ""
   },
+  {
+    "Name": "<a href=\"http://purl.org/pan-science/ESRFET#XRAYS\">XRAYS</a>",
+    "Alternative names": "X-ray Radiation Technique",
```

Comment:
Should we add a relation between the existing techniques and XRAYS, like a technique is `subClassOf` `XRAYS`?

Reply:
The relationship is _inferred_ as opposed to _asserted_.

For example this as asserted  _asserted_

- "http://purl.org/pan-science/ESRFET#XRAYS" is defined as "an experimental technique that requires X-rays as input". 
- "http://purl.org/pan-science/ESRFET#XRPD" is defined as "an experimental technique that requires X-rays as input, requires the sample to be a powder and measures the result of diffraction of elastic scattering".

From this this the reasoner understands that XRPD is a special case of XRAYS.